.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
```suggestion
* text eol=crlf
```

Reply:
Can't see why not, though it may flag up changes on a bunch of files as these were all previously ```lf```.

Reply:
i think the `lf` setting is what caused these non-change changes in your pr. 
![image](https://user-images.githubusercontent.com/1457540/120356914-b36dfc00-c2b9-11eb-8351-faa4a497758e.png)


not a big deal but it now looks like you've made tons of changes. so many that you made the top contributors on the website :sweat_smile: 

![image](https://user-images.githubusercontent.com/1457540/120357091-ec0dd580-c2b9-11eb-85b0-752a84fc55ea.png)

Reply:
Ah :/ 
That's slightly embarrassing. I only made the change as it was breaking the build process and the majority of the other files were LF anyway. By all means do what you need to do to drop me from that list haha

Reply:
haha. thanks for the link it buts all good.